clean/grand_isle_pd.py

```diff
@@ -135,3 +139,144 @@ def clean():
     df.to_csv(data_file_path(
         'clean/pprr_grand_isle_pd_2021.csv'
     ), index=False)
+<<<<<<< HEAD
+=======
```

Comment:
Again, unresolved conflicts.

Reply:
I've been only treating unresolved conflicts only when github informs me that there is a conflict. I'll find a better way to be informed. Will fix these now.